clients/drcachesim/drpt2trace/ir2trace.cpp

```diff
@@ -74,7 +80,14 @@ ir2trace_t::convert(IN drir_t &drir, INOUT std::vector<trace_entry_t> &trace)
             }
         } else {
 #ifdef DEBUG
-            ERRMSG(WARNMSG_HEADER "Try to convert an invalid instruction.\n");
+            warning_msg_mutex_.lock();
```

Comment:
Hmm.. rethinking this suggestion. Having a lock-unlock at each instr is a bit inefficient. How about printing these errors only if verbosity > some_threshold? And then set a lower threshold in tests?

Reply:
Done.